src/Header/Header.css

```diff
@@ -26,3 +26,29 @@
     padding-bottom: 6px;
 }
 
+.user{
+    display: inline-block;
+    position: absolute;
+    right: 0.5%;
```

Comment:
You almost always don't want to use percentages for positioning. It'll look weird on small/large screens usually (but I could be wrong)